Makefile

```diff
@@ -268,25 +268,7 @@ SUB_MAKE= `test -x ${MYMAKE} && echo ${MYMAKE} || echo ${MAKE}` \
 SUB_MAKE= ${MAKE} -m ${.CURDIR}/share/mk
 .endif
 
-.if defined(CHERI)
-TARGET=		mips
-TARGET_ARCH=	mips64
-CHERI_FLAGS=	-DDB_FROM_SRC \
-		-DMALLOC_PRODUCTION \
-		LOCAL_DIRS="ctsrd tools/tools/atsectl" \
```

Comment:
We should retire ctsrd too, the only stuff there is DE4 bits that I don't think have been used for many years even whilst DE4s were in use (though could be wrong about that last part)

Reply:
I pushed commits to remove these two dirs as these are the only references I could find to them.